spec/author_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './author'
+require './game'
+require './item'
+
+describe Author do
+  subject { described_class.new('Cal', 'Newport') }
+  describe '#add_item' do
+    it 'should add an item to the items array' do
+      game = Game.new(Date.new(2000, 1, 1), false, true, Date.new(2018, 1, 1))
+      subject.add_item(game)
+      expect(subject.items).to include(game)
+      expect(game.author).to eq(subject)
+      expect(game.author.first_name).to eq('Cal')
+    end
+  end
+end
```

Comment:
- The above test code looks reasonable. It tests the behavior of the Author class's add_item method to ensure that it adds an item to the items array. 
- My suggestion is to Improve the test code to add more test cases that cover the possible scenarios such as adding multiples items to the items array